ext/pycall/pycall.c

```diff
@@ -1013,6 +1016,9 @@ pycall_call_python_callable(PyObject *pycallable, int argc, VALUE *argv)
   }
 
   res = pyobject_call(pycallable, args, kwargs); /* New reference */
+  pycall_Py_DecRef(pycallable);
```

Comment:
I think this `DecRef` should be needless because the reference count of `pycallable` isn't increased in this function.

Reply:
Yes. This `DecRef(pycallable)` causes Seg. Fault and should be deleted.